doc/ipythonconsole.rst

```diff
@@ -2,91 +2,140 @@
+In addition, you can have each console display how long it has been running with :guilabel:`Show elapsed time`.
 
-==================
-Supported features
-==================
+.. image:: images/console/console-options-menu.png
+   :alt: Spyder IPython Console with options menu
 
-Any :guilabel:`IPython Console` in Spyder, internally or externally created, supports additional features including:
+You can also change the name of the current :guilabel:`IPython console` tab with the :guilabel:`Rename tab` option, or by simply double-clicking it.
 
-.. image:: images/console/console-completion.png
-   :alt: Spyder IPython Console, with a popup list of code completion guesses
+.. image:: images/console/console-rename.gif
+   :alt: Spyder IPython Console showing renaming console
 
-* Automatic code completion
-* Real-time function calltips
-* Debugging toolbar integration for launching the debugger and controlling execution flow
 
-Spyder-created consoles support even more advanced capabilities, such as:
 
-* The :doc:`variableexplorer`, with GUI-based editors for many built-in and third-party Python objects
-* Full GUI integration with the enhanced IPython debugger, ``ipdb``, including viewing and setting normal and conditional breakpoints interactively in any file, a :guilabel:`Breakpoints` pane, and following along with execution flow in the in the :doc:`editor` (see the :doc:`debugging` documentation for more details)
-* The :ref:`User Module Reloader <umr-section>`, which can automatically re-import modified packages and files
-* Inline display of Matplotlib graphics, if the ``Inline`` backend is selected under :menuselection:`Preferences --> IPython console --> Graphics --> Graphics backend`
+.. _connecting-external-kernel:
 
-For information on the features, commands and capabilities built into IPython itself, see the `IPython documentation`_.
+==================================
+Using external kernels with Spyder
+==================================
+
+You can connect to external local and remote kernels (including those managed by Jupyter Notebook or QtConsole) through the :guilabel:`Connect to an existing kernel` dialog under the :guilabel:`Consoles` menu.
+For this feature to work, a compatible version of the ``spyder-kernels`` package :ref:`must be installed <starting-kernel-problems-ref>` in the environment or machine in which the external kernel is running.
+
+.. image:: images/console/console-menu.png
+   :alt: Connect to external kernel dialog of the Spyder IPython console
+
+
+Connect to a local kernel
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To connect to a local kernel that is already running (e.g. one started by Jupyter notebook),
+#. Run ``%connect_info`` in the notebook or console you want to connect to, and copy the name of its kernel connection file, shown after ``jupyter <app> --existing``.
```

Comment:
```suggestion
To connect to a local kernel that is already running (e.g. one started by Jupyter notebook),

#. Run ``%connect_info`` in the notebook or console you want to connect to, and copy the name of its kernel connection file, shown after ``jupyter <app> --existing``.
```

It seems a line break is missing here because this looks incorrect on Netlify:

![Selección_078](https://user-images.githubusercontent.com/365293/102700043-d5962500-4217-11eb-911d-eb2d7d63236a.png)

Reply:
Yep, good catch. Probably an issue with Github suggestions not getting applied correctly when blank lines are involved.

Reply:
I'll make sure this is fixed when I push my revisions to this section momentarily.

Reply:
It was